.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📖 OpenFGA's Documentation
+    url: https://openfga.dev/docs/authorization-and-openfga
```

```suggestion
    url: https://openfga.dev/
```
Should link to top page in case we add/remove/rename pages

---

Updated in [`87631cc` (#1262)](https://github.com/openfga/openfga/pull/1262/commits/87631cc931b355ce7fd557f34e7250ecbc3d5119)

Left the `/docs` as it's supposed to go directly to the docs page and not the home page.